packages/orbit-components/src/Popover/index.js

```diff
@@ -59,7 +84,7 @@ const Popover = ({
   const handleOut = React.useCallback(
     ev => {
       // If open prop is present ignore custom handler
-      if (container.current && !container.current.contains(ev.target)) {
+      if (ref && !ref.current.contains(ev.target)) {
```

Comment:
This happened because of wrong type above. `ref` will always exist, but `ref.current` is the one that needs to be checked because it might be `null`.
```suggestion
      if (ref.current && !ref.current.contains(ev.target)) {
```